api/v1beta1/condition_consts.go

```diff
@@ -50,5 +50,5 @@ const (
 	// Could not find the BMH associated with the Metal3Machine
 	MissingBMHReason = "MissingBMHReason"
```

Comment:
This also looks not accurate to me actually, should be i.e:

```suggestion
	MissingBMHReason = "MissingBMH"
```

so it is aligned with other reasons.

Reply:
@schrej would you like to fix this as well?